src/fusioncharts.component.ts

```diff
@@ -1,6 +1,6 @@
 import {Component, Input, ElementRef, OnInit,
     OnChanges, DoCheck, AfterViewInit, OnDestroy,
-    KeyValueDiffers, ViewChild} from '@angular/core';
+    KeyValueDiffers, ViewChild, NgZone} from '@angular/core';
```

Comment:
I forgot to put zone variable in the constructor because of another test.
NgZone is used in ngAfterViewInit().